<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidChassisShape</class>
 <widget class="QWidget" name="GuidChassisShape">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>shapeï¼š</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customComboBox" name="comboBox_shape">
          <item>
           <property name="text">
            <string>Rectangle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Please obtain the following robot parameters according to the product specification or design drawings. Click next.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Picture</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_image">
          <property name="text">
           <string>image</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_head">
            <property name="text">
             <string>head:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tail">
            <property name="text">
             <string>tail:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_width">
            <property name="text">
             <string>width:[m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_radius">
            <property name="text">
             <string>Contour radius:[m]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_head">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_tail">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_width">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="customDoubleSpinBox" name="doubleSpinBox_radius">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customComboBox</class>
   <extends>QComboBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
  <customwidget>
   <class>customDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
